modules/nft/Cargo.toml

```diff
@@ -17,6 +17,7 @@ pallet-proxy = { git = "https://github.com/paritytech/substrate", branch = "polk
 primitives = { package = "acala-primitives", path = "../../primitives", default-features = false }
 sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.8", default-features = false }
 sp-std = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.8", default-features = false }
+chainbridge = { git = "https://github.com/AcalaNetwork/chainbridge-substrate", default-features = false, optional = true }
```

Comment:
why nft need chainbridge?

Reply:
It's only if you run benchmark tests inside directory that's why it's optional.

Reply:
but that's wrong, it is something else need chainbridge, not nft

Reply:
removed